src/core/dbus-execute.c

```diff
@@ -1770,6 +1783,21 @@ int bus_exec_context_set_transient_property(
 
                 return 1;
 
+        } else if (streq(name, "CPUAffinityFromNUMA")) {
+                int q;
+
+                r = sd_bus_message_read_basic(message, 'b', &q);
+                if (r < 0)
+                        return r;
+
+                if (!UNIT_WRITE_FLAGS_NOOP(flags))
+                        if (q) {
```

Comment:
Hmm, I think this needs to be written unconditionally. It is possible that the setting is set to true somewhere in existing configuration, and then setting it to false needs to be written to override that.